### DISCUSSION TITLE
1v1.lol

### DISCUSSION THREAD

**Discussion by giscus** [astrogames06/astrogames06.github.io#1] [Q&A]
# 1v1.lol

http://127.0.0.1:5500/game.html

**Comment by astrogames06** [comment]
good game

**Reply by astrogames06** [reply]
i agree

**Reply by astrogames06** [reply]
i also agree

**Reply by ur-mom847** [reply]
my game will not load :/

**Comment by astrogames06** [comment]
![Dancing](https://media.tenor.com/DLVpLKd2Z6AAAAAM/black-security-guard-dancing-black-guard-meme.gif)

**Reply by 584997-ai** [reply]
<img src="https://i.makeagif.com/media/9-28-2023/Ruj-1j.gif" alt="Man Fighting Bear for Tastiest most tender Salmon TV Commercial John West |  2000&#39;s UK TV ADVERT on Make a GIF"/>

**Reply by 584997-ai** [reply]
Just watch the whole thing

[a comment by an automated account is not shown]
